converter/src/main/java/gov/cms/qpp/conversion/decode/QrdaDecoderEngine.java

```diff
@@ -263,7 +263,7 @@ private QrdaDecoder getDecoder(TemplateId templateId) {
 			}
 
 			Decoder decoder = qppDecoder.getClass().getAnnotation(Decoder.class);
-			TemplateId template = decoder == null ? TemplateId.DEFAULT : decoder.value();
+			TemplateId template = decoder.value();
```

Comment:
Are we assured that `decoder` will always be non-null here?

Reply:
I believe I only manually checked this once with a random template ID and did not receive null here. I'll double check to ensure, otherwise. I can make it pass `TemplateId.Unimplemented`

Reply:
`decoder` can be `null` only if the `QrdaDecoder` class isn't annotated with `@Decoder`.  Highly unlikely, especially that the decoder wouldn't be in the registry if it wasn't annotated.  But, to be safe, if `decoder` is `null`, we can return `null` from this method.